packages/core/styles/_typography.scss

```diff
@@ -153,6 +153,7 @@ h6,
 }
 
 p,
+.nhsuk-body,
```

Comment:
I wonder if adding `.nhsuk-body` should be considered separately, as part of a wider discussion over whether we require or support classes on regular html elements like `<p>` and `<a>` and `<ul>`?

Reply:
I think postpone for now? Feels unrelated to the rest of the PR?